ods_ci/tests/Tests/650__distributed_workloads/test-distributed-workloads-metrics-ui.robot

```diff
@@ -0,0 +1,117 @@
+    ...                the navigation menu on the left and page contents
+    [Tags]    RHOAIENG-4837
+    ...       Sanity    DistributedWorkloads
+    Open Distributed Workload Metrics Home Page
+    Wait until Element is Visible    ${DISTRIBUITED_WORKLOAD_METRICS_TEXT_XP}   timeout=20
+    Page Should Contain Element     ${DISTRIBUITED_WORKLOAD_METRICS_TITLE_XP}
+    Page Should Contain Element     ${DISTRIBUITED_WORKLOAD_METRICS_TEXT_XP}
+    Page Should Contain Element     ${PROJECT_XP}
+    Page Should Contain Element     ${PROJECT_METRICS_TAB_XP}
+    Page Should Contain Element     ${WORKLOAD_STATUS_TAB_XP}
+    Click Element    ${REFRESH_INTERNAL_MENU_XP}
+    ${get_refresh_interval_list}=    Get All Text Under Element   xpath=//*[starts-with(@id, "select-option-")]
+    Lists Should Be Equal    ${REFRESH_INTERNAL_LIST}    ${get_refresh_interval_list}
+
+Verify Project Metrics Default Page contents
+    [Tags]    RHOAIENG-4837
+    ...       Sanity    DistributedWorkloads
+    [Documentation]    Verifiy Project Metrics default Page contents
+    Open Distributed Workload Metrics Home Page
+    Select Distributed Workload Project By Name    ${PRJ_TITLE}
+    Check Project Metrics Default Page Contents
+
+Verify Distributed Workload status Default Page contents
+    [Tags]    RHOAIENG-4837
+    ...       Sanity    DistributedWorkloads
+    [Documentation]    Verifiy distributed workload status page default contents
+    Open Distributed Workload Metrics Home Page
+    Select Distributed Workload Project By Name    ${PRJ_TITLE}
+    Check Distributed Workload Status Page Contents
+
+Verify That Not Admin Users Can Access Distributed workload metrics default page contents
+    [Documentation]    Verify That Not Admin Users Can Access Distributed workload metrics default page contents
+    [Tags]    RHOAIENG-4837
+    ...       Tier1    DistributedWorkloads
+    Launch Dashboard    ocp_user_name=${TEST_USER_3.USERNAME}    ocp_user_pw=${TEST_USER_3.PASSWORD}
+    ...    ocp_user_auth_type=${TEST_USER_3.AUTH_TYPE}    dashboard_url=${ODH_DASHBOARD_URL}
+    ...    browser=${BROWSER.NAME}    browser_options=${BROWSER.OPTIONS}
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE_NONADMIN}   description=${PRJ_DESCRIPTION}
+    Open Distributed Workload Metrics Home Page
+    Select Distributed Workload Project By Name    ${PRJ_TITLE_NONADMIN}
+    Check Project Metrics Default Page Contents
+    Check Distributed Workload Status Page Contents
+    [Teardown]    Run Keywords
+    ...    Delete Data Science Project   ${PRJ_TITLE_NONADMIN}
+    ...    AND
+    ...    Wait Until Data Science Project Is Deleted  ${PRJ_TITLE_NONADMIN}
+    ...    AND
+    ...    Switch Browser    1
+
+
+*** Keywords ***
+Project Suite Setup
+    [Documentation]    Suite setup steps for testing Distributed workload Metrics UI
+    Set Library Search Order    SeleniumLibrary
+    Skip If Component Is Not Enabled    kueue
```

Comment:
Does it mean that whole test suite is skipped or just setup?

Reply:
Whole test suite is skipped.. I added this because DW Ui is disabled if kueue is `Removed`